docs/guides/deployment/docker.rst

```diff
@@ -0,0 +1,361 @@
+.. _ref_guide_deployment_docker:
+
+======
+Docker
+======
+
+When to use the `edgedb/edgedb`_ Docker image
+=============================================
+
+.. _edgedb/edgedb: https://hub.docker.com/r/edgedb/edgedb
+
+This image is primarily intended to be used directly when there is a
+requirement to use Docker containers, such as in production, or in a
+development setup that involves multiple containers orchestrated by Docker
+Compose or a similar tool. Otherwise, using the :ref:`ref_cli_edgedb_server`
+CLI on the host system is the recommended way to install and run EdgeDB
+servers.
+
+
+How to use this image
+=====================
+
+The simplest way to run the image (without data persistence) is this:
+
+.. code-block:: bash
+
+   $ docker run --name edgedb -d \
+       -e EDGEDB_SERVER_PASSWORD=secret \
```

Comment:
This is no longer "the simplest way", `-e EDGEDB_SERVER_SECURITY=insecure_dev_mode` is.